Basic-Car-Maintenance/Shared/Dashboard/Views/EditEventDetailView.swift

```diff
@@ -0,0 +1,79 @@
+//
+//  EditEventDetailView.swift
+//  Basic-Car-Maintenance
+//
+//  Created by Aaron Wilson on 10/5/23.
+//
+
+import SwiftUI
+
+struct EditMaintenanceEventView: View {
+    @Binding var selectedEvent: MaintenanceEvent
+    @ObservedObject var viewModel: DashboardViewModel
+    @State private var title = ""
+    @State private var date = Date()
+    @State private var notes = ""
+    @Environment(\.dismiss) var dismiss
+    
+    var body: some View {
+        NavigationStack {
+            Form {
+                Section {
+                    TextField("Title",
+                              text: $title)
+                } header: {
+                    Text("Title")
+                }
+                
+                DatePicker(selection: $date, displayedComponents: .date) {
+                    Text("Date")
+                }
+                
+                Section {
+                    TextField("Notes", text: $notes, prompt: Text("Additional Notes"), axis: .vertical)
+                } header: {
+                    Text("Notes")
+                }
+            }
+            .onAppear(perform: {
+                setMaintenanceEventValues(event: selectedEvent)
+            })
+            .navigationTitle(Text("Add Maintenance"))
+            .toolbar {
+                ToolbarItem(placement: .topBarTrailing) {
```

Comment:
These should be switched, the top trailing should be the update and the top leading should be the cancel

And put a single line break between the `ToolbarItem`s, so they're not right next to each other